### ISSUE TITLE
mycelo load-bot breaks under heavy load

### ISSUE THREAD

**Issue by piersy** [celo-org/celo-blockchain#1455]
## Expected Behavior

The load-bot should not fail when submitting many transactions

## Current Behavior

The load bot fails after a few seconds with when a high tps is selected
```
./build/bin/mycelo load-bot --tps 100 loadtest
Error sending transaction: already known
Error sending transaction: failed to retrieve account nonce: context canceled
Error sending transaction: failed to estimate gas needed: context canceled
Error sending transaction: failed to retrieve account nonce: context canceled
Error sending transaction: failed to estimate gas needed: context canceled
Error sending transaction: failed to estimate gas needed: context canceled
Error sending transaction: failed to estimate gas needed: context canceled
Error sending transaction: failed to estimate gas needed: context canceled
Error sending transaction: failed to estimate gas needed: context canceled
Error sending transaction: failed to estimate gas needed: context canceled
Error sending transaction: failed to estimate gas needed: context canceled
Error sending transaction: failed to estimate gas needed: context canceled
Error sending transaction: already known
```

## Steps to Reproduce Behavior

Start a network with mycelo and run a load bot with a high tps.

E.G `./build/bin/mycelo load-bot --tps 100 loadtest`

## System information

Run `geth version`
```
Celo
Version: 1.0.0-unstable
Git Commit: 42915a56d74073c33dce07ced6c3cf324990b242
Git Commit Date: 20210325
Architecture: amd64
Protocol Versions: [66 65 64]
Go Version: go1.13.8
Operating System: linux
GOPATH=/home/pierspowlesland/go
GOROOT=/usr/local/go
```

**Comment by trianglesphere** [comment]
I was able to reproduce this on `60c3de2466a3d532bd8a0b188dcf2c9640a62e74` (a couple commits behind master as the load bot does not do replay protection). It looks like the issue is running a high tps load bot test against the default mycelo template.

My suspicion is that the default template only faucets 10 accounts which makes it hard for the load bot to run multiple transactions from those accounts in the same block.

Script to repro.
```
#!/usr/bin/env bash
enable -n kill
mc=./build/bin/mycelo
geth=./build/bin/geth

tps=100
txlim=500
ENVDIR=datadir
rm -rf $ENVDIR

# Causes issues
# $mc genesis --newenv "$ENVDIR" 

# Does not cause issues
$mc genesis --newenv "$ENVDIR" --template loadtest &> /dev/null

$mc validator-init --geth $geth  "$ENVDIR" &> /dev/null

$mc validator-run "$ENVDIR" --geth $geth 2> validator.log &
testnetPID=$!
echo Started the validator $testnetPID

sleep 5
$mc --verbosity=3 load-bot "$ENVDIR" --tps $tps --maxpending $txlim & 
loadPID=$!
echo started the load bot $loadPID

sleep 60
kill "$loadPID" 2> /dev/null
kill "$testnetPID" 2> /dev/null
```

**Comment by piersy** [comment]
@trianglesphere Thanks for this, please see the fix at #1456
